public/app/features/admin/UserInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { User } from 'app/types';
+
+interface Props {
+  user: User;
+  className: string;
+}
+
+export const UserInfo: FC<Props> = ({ className, user }) => {
+  return (
+    <>
+      <table className={`${className} filter-table form-inline`}>
```

Comment:
for optional & conditional classes use cx function from emotion or classnames util